db/migrate/20200515225546_create_solution_mentor_discussion_posts.rb

```diff
@@ -7,6 +7,7 @@ def change
 
       t.text :content_markdown, null: false
       t.text :content_html, null: false
+      t.string :uuid, null: false
```

Comment:
This will need to be a separate migration as we're not resetting the staging environment on deploy anymore.